docs/source/api/bounds.rst

```diff
@@ -66,7 +66,7 @@ Caveats
 
 * Bounds cannot be given to variables that are ``observed``.  To model
   truncated data, use a :func:`~pymc3.model.Potential` in combination with a cumulative
-  probability function.  See `this example notebook <https://github.com/pymc-devs/pymc3/blob/master/docs/source/notebooks/weibull_aft.ipynb>`_.
+  probability function.  See `this example notebook <https://github.com/pymc-devs/pymc-examples/blob/main/examples/survival_analysis/weibull_aft.ipynb>`_.
```

Comment:
shouldn't this point to the website instead of github?

Reply:
```suggestion
  probability function.  See `this example notebook <https://docs.pymc.io/pymc-examples/examples/survival_analysis/weibull_aft.html>`_.
```